### PR TITLE
Creates Custom WebViewDialog to present WebView

### PR DESCRIPTION
**Pull request by Ariflo**

### CODE REVIEW

File: magic/core/src/main/java/link/magic/android/core/relayer/WebViewWrapper.kt [hunk 1]
```diff
@@ -38,6 +37,7 @@ class WebViewWrapper internal constructor(context: Context, private val urlBuild
 
     internal val mMutableContext = MutableContextWrapper(context)
     private val webView: WebView = WebView(mMutableContext)
+    private var webViewDialog: WebViewDialog? = null
```
Comment (Ariflo) [inline magic/core/src/main/java/link/magic/android/core/relayer/WebViewWrapper.kt:40]: I am not a fan of how I'm initializing the `WebViewDialog` here, but alas my Android achilles reveals it self. 😞 

As always, open to any suggestion you might have here.

File: magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import android.app.Dialog
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.webkit.WebView
+import android.widget.RelativeLayout
+
+class WebViewDialog(context: Context, private val webView: WebView) : Dialog(context){
```
Comment (Ethella) [inline magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt:11]: Can this context be ApplicationContext? Or it has to be an Activity context?

File: magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        window?.setBackgroundDrawableResource(android.R.color.transparent)
+        window?.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND)
+    }
+
+    private fun createContentView(): View {
+        val layout = RelativeLayout(context)
+        layout.addView(webView, ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT)
+        return layout
+    }
+}
```
Comment (Ethella) [inline magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt:25]: blank space

File: magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt (hunk 2 above)
Comment (Ariflo) [reply magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt:11]: I believe it can accept either. In this case, I'm making sure to pass it whichever context is passed by `runOnUiThread`

File: magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt (hunk 3 above)
Comment (Ariflo) [reply magic/core/src/main/java/link/magic/android/core/relayer/WebViewDialog.kt:25]: I believe this line has been removed and GitHub is just showing it here.

File: example/build.gradle [hunk 4]
```diff
@@ -42,12 +42,12 @@ dependencies {
     testImplementation "junit:junit:$junit_version"
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    implementation project(':magic:core')
-    implementation project(':magic:ext:oauth')
-    implementation project(':magic:ext:oidc')
-//    implementation 'link.magic:magic-android:[6.0,7.0['
```
Comment (Ethella) [inline example/build.gradle:48]: Can we revert this change?

**Review by Ethella — APPROVED** [review]
